src/styles/_colors.scss

```diff
@@ -0,0 +1,4 @@
+$primary-color: hsl(47, 88%, 63%);
```

Comment:
we rarely use hsl colors, we usually use rgb ( rgba ) or hex values as those are the ones that are used within designs

Reply:
those are taken from the `style-guide`